Jenkinsfile

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/groovy
+@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+def dummy
+dockerTemplate{
+  dockerNode{
```

Comment:
why ? there isn't anything related to go here... Maybe it requires another template but we just run nginx inside.